examples/nextjs-full-stack/README.md

```diff
@@ -1,64 +1,94 @@
-## About the app
+# Webcam Screenshot Next.js Full Stack App Example
 
-This full-stack app shows how to build a Next.js application that uploads a camera image to a Miro board by using the Miro Web SDK and Miro REST API. This app demonstrates the use of the Miro REST API and Web SDK. After configuring your project outlined in the Prerequisites section, this app allows you to authenticate yourself with your Miro account, and use the related access tokens to make REST API calls via the Miro Developer Platform.
+This full-stack app shows how to build a Next.js application that uploads a camera image to a Miro board by using the Miro Web SDK and Miro REST API.
 
-This app also stores your access and refresh tokens in your browsers cookies to demonstrate the Miro's REST API. When storing tokens in your own app, you should be consider encrypting or storing your user's tokens in a more secure way.
+# 👨🏻‍💻 App Demo
 
-## Stack
+https://github.com/miroapp/app-examples/assets/10428517/1bc9e8e0-48bd-43c2-8ba2-cf181ce58eee
 
-- [x] [React](https://reactjs.org/)
-- [x] [Next.js](https://nextjs.org/)
-- [x] [Miro Node.js API client](https://developers.miro.com/docs/miro-nodejs-readme)
-- [x] [Mirotone](https://www.mirotone.xyz/css) (styling)
+# 📒 Table of Contents
 
-## Prerequisites
+- [Included Features](#features)
+- [Tools and Technologies](#tools)
+- [Prerequisites](#prerequisites)
+- [Associated Developer Tutorial](#tutorial)
+- [Deploy the app on AWS Amplify](#deploy)
+- [Run the app locally](#run)
+- [Folder Structure](#folder)
+- [Contributing](#contributing)
+- [License](#license)
 
-1. Create a [Developer team in Miro](https://developers.miro.com/docs/create-a-developer-team).
-2. Create an [app in Miro](https://developers.miro.com/docs/build-your-first-hello-world-app#step-1-bootstrap-the-hello-world-app).
-3. Create a board in Miro that you'd like to manipulate with the REST API.
+# ⚙️ Included Features <a name="features"></a>
 
-## How to start
+- [Miro Node.js Client Library](https://developers.miro.com/docs/web-sdk-reference)
+  - [miro.isAuthorized](https://miroapp.github.io/api-clients/classes/index.Miro.html#isAuthorized)
```

Comment:
Are we exposing the github pages url ?

Reply:
how come it's not part of the dev portal ?

Reply:
I will ping you separately. This in unrelated to the PR. :D

Reply:
This is just where the Node.js client is hosted right @imxena ?

I thought it shouldn't be an issue, but please let me know if I am wrong here. I just wanted to point them in the direction of the docs for the API methods which are used for this example.

Reply:
@horeaporutiu  I was surprised cz I also saw the github page url exposed directly for the first time & thought that it should have been a developer portal (developers.miro.com) url. It is indeed the url to share for api client docs for now.